woocommerce/woocommerce-android
# [Tablet] Fix Order Details Infinite Shimmer Issue

Closes: #10881


### Description

Shows an empty view for Order Details when no Orders exists. 

### Testing instructions
1. Use a store with no orders. 
2. Go to the orders tab. 
3. Observe the Empty View to the right. 

**N.B** The Order List currently shows the filters and the notice card. That is outside the scope of this issue but could be addressed here. Nonetheless in the interest of time and not having this shimmer effect this improvement could be shipped and the optimizations ticketed for further fixing. 

**Color**: I took the best approach I could think of standardizing the color across the products and orders. A better approach may exist but I couldn't find any in the moment. 



### Images/gif


<kbd><img src="https://github.com/woocommerce/woocommerce-android/assets/1509205/b752ffe5-9172-4985-b7f8-65480eb9e881" width="640"></kbd>
